custom_components/localtuya/climate.py

```diff
@@ -250,6 +264,23 @@ def hvac_action(self):
                 ) > self._target_temperature:
                     self._hvac_action = CURRENT_HVAC_IDLE
             return self._hvac_action
+            if self._hvac_mode == HVAC_MODE_COOL:
+                if self._current_temperature < (
```

Comment:
Is the sign the right way around? I think it makes sense re heating (if less than target temp, heat); but not so much cooling